### ISSUE TITLE
Default rust test creates 1,000,000 test tokens in the setup function

### ISSUE THREAD

**Issue by SwayStar123** [FuelLabs/sway#2605]
```
async fn get_contract_instance() -> (MyContract, ContractId) {
    // Launch a local network and deploy the contract
    let mut wallets = launch_custom_provider_and_get_wallets(
        WalletsConfig::new(Some(1), Some(1000000), Some(1_000_000)),
        None,
    )
    .await;
    let wallet = wallets.pop().unwrap();

    let id = Contract::deploy(
        "./out/debug/ocr.bin",
        &wallet,
        TxParameters::default(),
        StorageConfiguration::with_storage_path(Some(
            "./out/debug/ocr-storage_slots.json".to_string(),
        )),
    )
    .await
    .unwrap();

    let instance = MyContractBuilder::new(id.to_string(), wallet).build();

    (instance, id.into())
}
```

This is the current default setup function generated by forc, it generates 1,000,000 test tokens for the wallet. This slows down the tests  by minutes. Should be lowered to 20 or so

**Comment by simonr0204** [comment]
This is more for [fuels-rs](https://github.com/FuelLabs/fuels-rs) I would think.

**Comment by SwayStar123** [comment]
> This is more for [fuels-rs](https://github.com/FuelLabs/fuels-rs) I would think.

the default test is generated by forc no?

**Comment by simonr0204** [comment]
> > This is more for [fuels-rs](https://github.com/FuelLabs/fuels-rs) I would think.
> 
> the default test is generated by forc no?

Yes, I got momentarily confused and thought this was an SDK helper function - deleted my comment but not fast enough it seems!

**Comment by kayagokalp** [comment]
Closing this issue as this is solved by #2573 and probably provided example is created before the latest version of `forc` If you feel like this is still relevant please re-open 👍
